packages/core/src/Chip/ChipContent/ChipContent.tsx

```diff
@@ -39,7 +40,7 @@ export type ChipContentProps = BoxProps & {
     title?: React.ReactNode
   }
   image?: React.ReactNode
-  size?: string
+  size?: ResponsiveValue<'sm' | 'md'> | 'sm' | 'md'
```

Comment:
From the types perspective, `ResponsiveValue<'sm' | 'md'>` might be able to handle `'sm' | 'md'` as well